test/fixtures/dialects/postgres/postgres_create_database.sql

```diff
@@ -0,0 +1,28 @@
+CREATE DATABASE db;
+CREATE DATABASE db OWNER user_name;
+CREATE DATABASE db OWNER = user_name;
+CREATE DATABASE db WITH OWNER = user_name;
+CREATE DATABASE db TEMPLATE template_name;
+CREATE DATABASE db TEMPLATE = template_name;
+CREATE DATABASE db WITH TEMPLATE = template_name;
+CREATE DATABASE db ENCODING 'UTF8';
+CREATE DATABASE db ENCODING = 'UTF8';
+CREATE DATABASE db LOCALE 'en_US.UTF-8';
+CREATE DATABASE db LOCALE = 'en_US.utf8';
+CREATE DATABASE db LC_COLLATE 'en_US.UTF-8' LC_CTYPE 'en_US.UTF-8';
+CREATE DATABASE db WITH LC_COLLATE 'en_US.UTF-8' LC_CTYPE 'en_US.UTF-8';
+CREATE DATABASE db LC_COLLATE = 'en_US.UTF-8' LC_CTYPE = 'en_US.UTF-8';
+CREATE DATABASE db WITH LC_COLLATE = 'en_US.UTF-8' LC_CTYPE = 'en_US.UTF-8';
+CREATE DATABASE db TABLESPACE DEFAULT;
+CREATE DATABASE db TABLESPACE = DEFAULT;
+CREATE DATABASE db TABLESPACE new_tablespace;
+CREATE DATABASE db TABLESPACE = new_tablespace;
+CREATE DATABASE db WITH TABLESPACE = new_tablespace;
+CREATE DATABASE db ALLOW_CONNECTIONS true;
+CREATE DATABASE db ALLOW_CONNECTIONS = true;
+CREATE DATABASE db CONNECTION LIMIT 10;
+CREATE DATABASE db CONNECTION LIMIT = 10;
+CREATE DATABASE db IS_TEMPLATE true;
+CREATE DATABASE db IS_TEMPLATE = true;
+CREATE DATABASE db IS_TEMPLATE = true CONNECTION LIMIT 10 ALLOW_CONNECTIONS = true;
+CREATE DATABASE db WITH IS_TEMPLATE = true CONNECTION LIMIT 10 ALLOW_CONNECTIONS = true;
```

Comment:
Wonder if we should have one test case with all of the options specified to really test the grammar?
It should work but good to check 😄

Reply:
Will do